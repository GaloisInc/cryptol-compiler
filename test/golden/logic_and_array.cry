[0xF0, 0x0F] && [0xD0, 0x00]