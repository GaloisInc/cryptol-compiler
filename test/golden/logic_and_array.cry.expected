[0xd0, 0x00]

